packages/models-tree/src/models-tree/ModelsTreeDefinition.ts

```diff
@@ -711,15 +712,25 @@ function createGeometricElementInstanceKeyPaths(
   );
 }
 
-function parseQueryRow(row: ECSqlQueryRow, groupInfos: ElementsGroupInfo[], separator: string) {
+function parseQueryRow(row: ECSqlQueryRow, groupInfos: ElementsGroupInfo[], separator: string, eClassName: string) {
```

Comment:
I'd rename `eClassName` to `elementClassName`

Reply:
Renamed